client/checkout/utils/platform-checkout.js

```diff
@@ -0,0 +1,28 @@
+// Check whether user is a valid Platform Checkout user
+export const handlePlatformCheckoutEmailInput = ( container, field ) => {
+	let timer;
+	const waitTime = 500;
+	const platformCheckoutEmailContainer = document.querySelector( container );
+
+	const platformCheckoutLocateUser = () => {
+		platformCheckoutEmailContainer.classList.add( 'is-loading' );
+
+		// Placeholder to simulate request. Replace with request to Platform Checkout email verification endpoint.
+		setTimeout( () => {
+			platformCheckoutEmailContainer.classList.remove( 'is-loading' );
+		}, 3000 );
+	};
+
+	const platformCheckoutEmailInput = document.querySelector( field );
+
+	platformCheckoutEmailInput.addEventListener( 'keyup', ( e ) => {
```

Comment:
I think the event we want to listen to is the `input` event. keyup will not be triggered on mobile or if inserting content with autocomplete prompts and other ways. `input` should always work. 
```suggestion
	platformCheckoutEmailInput.addEventListener( 'input', ( e ) => {
```

Reply:
👍 Thanks for catching this! Updated.